contracts/test/MockERC1155.sol

```diff
@@ -0,0 +1,22 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
As I understand this file is from the foundry-tests directory. Not critical at all, but will be good to move to a separate directory, like `contracts/test/foundry-tests` for example. Here is a lot of contracts already, hard to navigate. The same can be done for the Mock ERC20